app/controllers/monthly_reports_controller.rb

```diff
@@ -41,6 +44,9 @@ def update
     assign_relational_params(@monthly_report)
 
     if @monthly_report.save
+      if @monthly_report.status == "shipped"
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)